src/plugins/cachefilter/README.md

```diff
@@ -0,0 +1,16 @@
+- infos = Information about the cachefilter plugin is in keys below
+- infos/author = Marvin Mall <e1225943@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements =
+- infos/status = experimental global
```

Comment:
Various status information missing, please look through doc/CONTRACT.ini and add everything that fits.